force-app/main/default/classes/ESA_cls_CaseUpdate.cls

```diff
@@ -0,0 +1,28 @@
+/** 
+ * @author: accenture_sat_team
+ * @description: Created to perform the case update step action and invoked form Step Action apex calss
```

Comment:
This was called out in the meeting yesterday to explain how this class will be invoked. Will it be from Case Trigger Handler or Case record triggered flow or Case Quick action button. But based on the review now, i don't see the change here. Just wanted to callout and will conditionally approve due to the time.

Reply:
@cgiSunilM  Updated the description with all the mentioned details. Thanks !!